### ISSUE TITLE
which trained model is the most accurate

### ISSUE THREAD

**Issue by Zju-George** [1adrianb/face-alignment#267]
<img width="762" alt="image" src="https://user-images.githubusercontent.com/33959089/115148403-fec5a580-a091-11eb-91cc-60f197578bb9.png">
Silly question here but I really want to get higher accuracy.

**Comment by 1adrianb** [comment]
The weights of each group are identical and they simply reflect the internal changes made inside pytorch. Beyond this 2DFAN will produce 2D landmarks, 3DFAN - 2.5D landmarks and 3DFAN + the depth model will produce 3D landmarks. The projection of the 3D landmarks into 2D correspond with the 2.5D predictions.

**Comment by Zju-George** [comment]
@1adrianb Thanks for your reply! Yeah, I've just tested it out and I see that. 

Also, one thing I've noticed is that the eye landmarks being not aligned perfectly when the eye is closed.
Do you have any plan to fine-tune the model on images where people closing their eyes?

**Comment by 1adrianb** [comment]
Sorry for delay, yes, but I don't have an exact timeline unfortunately.

**Comment by Zju-George** [comment]
> Sorry for delay, yes, but I don't have an exact timeline unfortunately.

@1adrianb If you could share your training code, I would be willing to try the fine-tune work.
